scss/samples/_colors.scss

```diff
@@ -92,6 +92,28 @@ $demo-colors: (
   'brand-pink': $brand-pink,
 );
 
+/// ## CSS custom properties
+///
+/// At OddBird, we convert color tokens into
+/// CSS custom properties using [Accoutrement][ac].
+/// Color values stored as custom properties
+/// can also be previewed as palettes.
+///
+/// [ac]: https://www.oddbird.net/accoutrement
+///
+/// @colors
+/// @group demo_colors
+$custom-property-colors: (
+  'accent': var(--callout),
+  'link': var(--action),
+  'btn-text': var(--contrast-light),
```

Comment:
Huh, the result here is a bit confusing, but technically accurate. Maybe needs a bit of thought about what we're actually trying to show here. 

- Technically, there's a token named `accent`, and it has a value of `var(--callout)`. That's all displayed accurately. But…
- When we use a setup like this, we expect to generate a custom property for each token. If you want to use the `accent` token, you call `var(--accent)`. The current output confused me at first, because `--accent` isn't used anywhere.

I don't know if/how we can or should represent that. At this point, Herman doesn't know that `var(--accent)` will be generated, it only sees a token with a value - which we are displaying correctly.

I'd be interested in @stacyk's thoughts on this. At the minimum, we might want some more text that explains what's happening here?

Reply:
@stacyk @mirisuzanne 

Would it clear things up if I explain that the keys ('accent', 'link', 'btn-text') are example tokens based on tokens that were turned into custom properties in a separate location? 

My intention was to show previews from a map that uses uses custom properties instead of raw color values. Would it be less confusing if I used custom properties of tokens that we are already using on the page (i.e. `brand-blue`, `brand-pink`)?

Reply:
I think that using the brand colors that were defined earlier on this page provides clarity. I don't think the name you give them is something we have to be using necessarily:

![Screen Shot 2022-09-20 at 12 46 06 AM](https://user-images.githubusercontent.com/1581694/191176859-cf5946b1-8e3f-4824-847f-5560e02ad9d9.jpg)

Although, I do think we darken our brand pink at some point so the example above is showing the darkened pink on bottom and the regular pink above which is confusing. 

**Retreat Review**

In general, I feel like reading through a few pages of the current documentation, I would love to see more examples or perhaps visuals/graphs of how some things work/look, etc. 

Would it be confusing or appropriate anywhere to show how we are defining custom properties as sass variables in their own partial without a map and provide examples of what we do to turn it into a map/json data for Herman? (Maybe that is more of an Accoutrement example?)

Earlier on this page, it says "Whether you store colors as individual variables or group them into a map, you can use Herman to document and display color palettes." Do we show any examples of storing individual colors and displaying the palettes?

Reply:
For clarity's sake, I used the `brand-colors` custom properties here (with the color difference explanation @stacyk ). I also tried to explain a bit more about all that's going on here. 

![image](https://user-images.githubusercontent.com/41588129/191270521-a9c8b5cc-8bdd-486d-8a0d-c270a97d9d6f.png)


Is this more clear?

Reply:
Much more clear, and good thoughts @stacyk. Would be good conversations for retreat.